cabal.project

```diff
@@ -30,6 +30,9 @@ packages:
   libs/set-algebra
   libs/vector-map
 
+program-options
```

Comment:
My understanding is that the typical policy among IOG repos is for each developer to add `-Werror` to their `cabal.project.local` file, if they prefer, instead of unilaterally putting it in the repository. I don't know for sure whether that's the case for `cardano-ledger`, but I would guess so.

Reply:
It is the opposite in ledger. We have `-Werror` enabled for all projects. If a developer wants to turn them back into warnings then he/she can do so in the `cabal.project.local`